frontend/src/containers/ViewDashboardAdmin.tsx

```diff
@@ -513,6 +556,16 @@ function ViewDashboardAdmin() {
         buttonType={t("PreparePublishingModalButton")}
         buttonAction={onPublishDashboard}
       />
+
+      <Modal
+        isOpen={isOpenCopyModal}
+        closeModal={closeCopyModal}
+        title={`${t("DashboardListing.Copy")} "${dashboard.name}"?`}
```

Comment:
Now that we are going over translations will be nice if we can refactor this to use i18n interpolation

trasnlation.json
```
"Copy":  "Copy {dashboardName}"
``` 

usage
```
title={t("Copy", { dashboardName: dashboard.name })}
```

Reply:
@mazenkh-amazon DashboardListing.